Dockerfile.updater-core

```diff
@@ -115,7 +115,6 @@ RUN gem install bundler -v $BUNDLER_V2_VERSION --no-document && \
  rm -rf /var/lib/gems/*/cache/* && \
  bundle config set --global build.psych --with-libyaml-source-dir=$DEPENDABOT_HOME/src/libyaml/yaml-$LIBYAML_VERSION && \
  bundle config set --local path 'vendor' && \
- bundle config set --local frozen 'true' && \
```

Comment:
This is the kind of change that, if it's necessary, it's best to be discussed in isolation as a separate PR, since it has nothing to do with NuGet.

Reply:
Sure thing, I'll split this out along with the other non-NuGet changes into separate PRs.

Reply:
I've pulled out this commit, but now every Docker image fails to build because the Gemspec.lock files aren't present.  I'm not very familiar with the Ruby ecosystem, but how does this feature work in a regular PR?  What are the steps necessary to avoid the build errors I'm seeing?

Reply:
Not sure if I understood. We use `frozen` since it's a good practice to ensure that `Gemfile` and `Gemfile.lock` are in sync, and `frozen` enforces that. Why do you need to remove this in the first place? Also, can I see the errors you are getting?